server/src/main/java/com/lunchtime/mapper/PersonMapper.java

```diff
@@ -0,0 +1,39 @@
+package com.lunchtime.mapper;
+
+import com.lunchtime.models.Person;
+import com.lunchtime.service.dto.PersonDto;
+import com.lunchtime.repository.PersonRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+@Component
+@RequiredArgsConstructor
+public class PersonMapper {
+    final PersonRepository personRepository;
+
+    public Person fromDtoToPerson(PersonDto personDto) {
+        Optional<Person> result = personRepository.findById(personDto.getId());
+        if (result.isPresent()) {
+            Person person = result.get();
+            person.setName(personDto.getName());
+            person.setPhoneNumber(personDto.getPhoneNumber());
+            person.setEmail(personDto.getEmail());
+            person.setPhotoUrl(personDto.getPhotoUrl());
+            return person;
+        }
+        return null;
+    }
+
+    public PersonDto fromPersonToDto(Person person) {
+        PersonDto personDto = new PersonDto();
+
+        personDto.setId(person.getId());
+        personDto.setName(person.getName());
+        personDto.setPhoneNumber(person.getPhoneNumber());
+        personDto.setEmail(person.getEmail());
+        personDto.setPhotoUrl(person.getPhotoUrl());
+        return personDto;
+    }
```

Comment:
I think we should change PersonMapper's logic. It shouldn't know about repository. Due to SOLID only service works with Repository. So, Mappers repsonsibility is just to map, to pass one object's value to another. Thats why if we need to map PersonDto to already existing Person, we should pass both PersonDto and Person to `fromDtoToPerson(PersonDto perstoDto, Person person)`